sh/install.sh

```diff
@@ -76,13 +95,13 @@ function install_python() {
     if [ $OPERATING_SYSTEM == "DEBIAN" ]; then
         echo "Installing Python using apt-get"
         apt-get update >> ${INSTALL_LOG} 2>&1
-        apt-get install python -y >> ${INSTALL_LOG} 2>&1
+        apt-get install python3 -y >> ${INSTALL_LOG} 2>&1
     elif [ $OPERATING_SYSTEM == "REDHAT" ]; then
         echo "Installing Python using yum"
-        yum install python -y >> ${INSTALL_LOG} 2>&1
+        yum install python3 -y >> ${INSTALL_LOG} 2>&1
```

Comment:
Same question here
```suggestion
        yum install python3 python3-pip -y >> ${INSTALL_LOG} 2>&1
```

Reply:
I just wanted the pip installation code in a single place. that works for both pip2 and pip3

Reply:
Thanks, @yogeshprasad! I was also thinking about situations where we might have `python` pointing to `python3.x` but `pip` pointing to `pip2.x` and vise versa, also there might be situations where, say `python3.8` and `python3.6` are both installed on the system and `python3` points to `python3.8`, but `pip3` pointing to `python3.6`'s `pip` and so on. We have to make sure we handle those cases correctly.